lua/powerups/server/shotgun.moon

```diff
@@ -0,0 +1,75 @@
+{get: getConf} = CFCPowerups.Config
+
+import ShotgunPowerupHandler from CFCPowerups.SharedHandlers
+
+util.AddNetworkString "CFC_Powerups-Shotgun-Start"
+util.AddNetworkString "CFC_Powerups-Shotgun-Stop"
+
+export ShotgunPowerup
+class ShotgunPowerup extends BasePowerup
+    @powerupID: "powerup_shotgun"
+
+    @powerupWeights:
+        tier1: 1
+        tier2: 1
+        tier3: 1
+        tier4: 1
+
+    new: (ply) =>
+        super ply
+
+        @ownerSteamID64 = @owner\SteamID64!
+
+        @duration = getConf "shotgun_duration"
+        @singleBulletsMin = getConf "shotgun_single_bullets_min"
+        @singleBulletsMax = getConf "shotgun_single_bullets_max"
+        @singleDamageMult = getConf "shotgun_single_damage_multiplier"
+        @singleSpreadMult = getConf "shotgun_single_spread_multiplier"
+        @singleSpreadAdd = getConf "shotgun_single_spread_add"
+        @multiBulletsMult = getConf "shotgun_multi_bullets_multiplier"
+        @multiDamageMult = getConf "shotgun_multi_damage_multiplier"
+
+        @timerName = "CFC-Powerups_Shotgun-#{@ownerSteamID64}"
+        @handler = ShotgunPowerupHandler ply, @singleBulletsMin, @singleBulletsMax, @singleDamageMult, @singleSpreadMult, @singleSpreadAdd, @multiBulletsMult, @multiDamageMult
+
+        @ApplyEffect!
+
+    ApplyEffect: =>
+        super self
+
+        timer.Create @timerName, @duration, 1, -> @Remove!
```

Comment:
I think you can

```suggestion
        timer.Create @timerName, @duration, 1, @Remove
```

Reply:
It'll end up calling the remove function without the self arg, so that won't work. The method with the arrow is already used throughout the repo.